api/CustomStateSet.json

```diff
@@ -67,14 +83,23 @@
       },
       "clear": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomStateSet/clear",
+          "spec_url": "https://wicg.github.io/custom-state-pseudo-class/#customstateset",
```

Comment:
Note, this follows the pattern of other set-likes, linking to the closest thing that shows the setlike. Note that `add` is an exception as there is a specific link to it in the spec.